src/factories/components/expand/GameFactoriesExpandActionIcon.tsx

```diff
@@ -1,30 +0,0 @@
-import { useStore } from '@/core/zustand';
-import { useGameFactoriesHasAnyCollapsed } from '@/games/gamesSlice';
-import { ActionIcon, Tooltip } from '@mantine/core';
-import {
-  IconArrowsDiagonal,
-  IconArrowsDiagonalMinimize2,
-} from '@tabler/icons-react';
-export interface IGameFactoriesExpandActionIconProps {}
-
-export function GameFactoriesExpandActionIcon(
```

Comment:
Could we keep it in order to think of a place where to add it in the Spreadsheet view? It's very useful there (with new icons as you suggested)

Reply:
I removed it, because it's only being used in the spreadsheet. I could add it back, but it would be confusing in the kanban and grid view.